sympy/geometry/line.py

```diff
@@ -1199,12 +1273,24 @@ def __new__(cls, *args, **kwargs):
                 return LinearEntity.__new__(cls, p1, p2, **kwargs)
 
     def contains(self, other):
-        """
-        Return True if `other` is on this Line, or False otherwise.
+        """Checks whether `other` is on Line or not.
```

Comment:
This shouldn't be about 'other'

Reply:
I think this is what a function is doing. It's checking whether the point is on it or not.